packages/opentelemetry-cloud-monitoring-exporter/src/transform.ts

```diff
@@ -119,6 +129,57 @@ export function createTimeSeries(
   };
 }
 
+function transformResource(
+  resource: Resource,
+  projectId: string
+): MonitoredResource {
+  const templateResource = getTypeAndMappings(resource);
+  const type = templateResource.type;
+  const labels: { [key: string]: string } = { project_id: projectId };
+
+  for (const key of Object.keys(templateResource.labels)) {
+    if (!resource.labels[templateResource.labels[key]]) {
+      return { type: 'global', labels: { project_id: projectId } };
```

Comment:
```suggestion
      return { type: 'global', labels };
```
You've already defined labels as { project_id: projectId}.

Reply:
I see that I might be wrong with my suggestion, given that you mutate labels. I'm opting to leave this comment instead as a question: is it your intention with this function to return only {type:'global', labels: {project_id: projectId}} if any of the labels are missing? I assume yes, but it's probably worth a function comment describing what the caller can expect if their resource is missing a key:value pair.